alot/db/utils.py

```diff
@@ -110,7 +110,7 @@ def message_from_file(handle):
 
     # handle OpenPGP signed data
     if (m.is_multipart() and
-        m.get_content_subtype() == 'signed' and
+            m.get_content_subtype() == 'signed' and
```

Comment:
I think if there is stuff after the opening brace then everything should align with the opening brace+1. That would mean that line 114 would have to take the old indent of 113 and not the other way round. Because that would result in exactly 4 spaces that's ugly because it aligns with the default indent of the inner block of the if.

Not really sure what to do here :/